Source/Core/VideoBackends/OGL/Render.cpp

```diff
@@ -1403,6 +1406,7 @@ void Renderer::SwapImpl(u32 xfbAddr, u32 fbWidth, u32 fbStride, u32 fbHeight,
           std::make_unique<FramebufferManager>(s_target_width, s_target_height, s_MSAASamples);
 
       PixelShaderManager::SetEfbScaleChanged();
+      BoundingBox::SetEFBScaleChanged(Renderer::EFBToScaledXf(1), Renderer::EFBToScaledYf(1));
```

Comment:
`Renderer::` can be omitted here, since it's part of this class via inheritance (#4999 will also eliminate static renderer state)